lib/contexsample_web/live/piechart.ex

```diff
@@ -2,42 +2,36 @@ defmodule ContexSampleWeb.PieChartLive do
   use Phoenix.LiveView
   use Phoenix.HTML
 
-  alias Contex.SimplePie
+  import ContexSampleWeb.Shared
+
+  alias Contex.PieChart
 
   def render(assigns) do
     ~L"""
-      <h3>Simple Pie Chart Example</h3>
+      <h3>Pie Chart Example</h3>
```

Comment:
This liveview will be used for normal PieChart example using Plot logic. The `SimplePieChart` logic is moved to `ontexSampleWeb.SimplePieChartLive` ... below.